js/src/tooltip.js

```diff
@@ -465,6 +448,30 @@ class Tooltip extends BaseComponent {
 
   // Private
 
+  _initialiseOnDelegatedTarget(event, context) {
+    const dataKey = this.constructor.DATA_KEY
+    context = context || Data.getData(event.delegateTarget, dataKey)
+
+    if (!context) {
+      context = new this.constructor(event.delegateTarget, this._getDelegateConfig())
+      Data.setData(event.delegateTarget, dataKey, context)
+    }
+
+    return context
+  }
+
+  _enableIosTouchEventsTrick() {
+    // If this is a touch-enabled device we add extra
+    // empty mouseover listeners to the body's immediate children;
+    // only needed because of broken event delegation on iOS
+    // https://www.quirksmode.org/blog/archives/2014/02/mouse_event_bub.html
+    if ('ontouchstart' in document.documentElement) {
+      [].concat(...document.body.children).forEach(element => {
+        EventHandler.on(element, 'mouseover', noop())
```

Comment:
Shouldn't this be just `noop`?

Reply:
sincerely I only moved it, as it was duplicated.  Didn't change it.

Reply:
I know :) But I think it should be `noop`. @rohit2sharma95 WDYT?

Reply:
I cross checked it, the one was 'noop', the other was 'noop()' (wrong). And I was wondering why the script was running with out reason the first time